root/static/scripts/guess-case/MB/Control/GuessCase.js

```diff
@@ -60,7 +61,7 @@ var mode = ko.computed({
 
         if (modeName !== gc.modeName) {
             gc.modeName = modeName;
-            gc.mode = MB.GuessCase.Mode[modeName];
+            gc.mode = require('../../modes')[modeName];
```

Comment:
http://jsmodules.io/cjs.html is also useful.

The biggest different between these module systems is, `import` statements are statically analyzable at compile-time. `require` is just a function executed at run-time like any other, so you can e.g. pass in a variable containing the path to a module, which just isn't possible with `import`.